### PR TITLE
docs(cn): translate docusaurus/docs/using-https-in-development.md into Chinese

### PR DESCRIPTION
**Pull request by fikyair**
using-https-in-development 翻译初稿

### CODE REVIEW

File: docusaurus/docs/using-https-in-development.md [hunk 1]
```diff
@@ -4,19 +4,19 @@ title: Using HTTPS in Development
 sidebar_label: HTTPS in Development
 ---
 
-> Note: this feature is available with `react-scripts@0.4.0` and higher.
+> 注意: 在react-scripts@0.4.0及更高版本中可以使用此功能
```
Comment (QC-L) [inline docusaurus/docs/using-https-in-development.md]:
```suggestion
> 注意：在 `react-scripts@0.4.0` 及更高版本中可以使用此功能
```

File: docusaurus/docs/using-https-in-development.md [hunk 2]
```diff
@@ -4,19 +4,19 @@ title: Using HTTPS in Development
...
 
-You may require the dev server to serve pages over HTTPS. One particular case where this could be useful is when using [the "proxy" feature](proxying-api-requests-in-development.md) to proxy requests to an API server when that API server is itself serving HTTPS.
+你可能需要使开发服务器支持https。有一种特殊情况是，当该API服务器本身支持HTTPS时，使用 [proxy](proxying-api-requests-in-development.md) 功能可以将请求代理到该API服务器。
```
Comment (QC-L) [inline docusaurus/docs/using-https-in-development.md]:
```suggestion
有时，你可能需要在开发环境中使用 HTTPS。有一种特殊情况，当该 API 服务器本身支持 HTTPS 时，使用 [proxy](proxying-api-requests-in-development.md) 功能可以将请求代理到此 API 服务器。
```

File: docusaurus/docs/using-https-in-development.md [hunk 3]
```diff
@@ -4,19 +4,19 @@ title: Using HTTPS in Development
...
+为此，将 `HTTPS` 环境变量设置为 `true` ，然后像往常一样使用 `npm start` 启动开发服务器：
 
 ### Windows (cmd.exe)
 
 ```cmd
 set HTTPS=true&&npm start
 ```
 
-(Note: the lack of whitespace is intentional.)
+(注意: 中间是没有空格的。)
```
Comment (QC-L) [inline docusaurus/docs/using-https-in-development.md]:
```suggestion
(注意: 中间没有空格。)
```

File: docusaurus/docs/using-https-in-development.md [hunk 4]
```diff
@@ -30,15 +30,15 @@ set HTTPS=true&&npm start
 HTTPS=true npm start
 ```
 
-Note that the server will use a self-signed certificate, so your web browser will almost definitely display a warning upon accessing the page.
+注意，服务器将使用自签名证书，因此web浏览器在访问页面时可能会显示警告。
```
Comment (QC-L) [inline docusaurus/docs/using-https-in-development.md]:
```suggestion
注意，服务器使用自签名证书，Web 浏览器在访问页面时可能会显示警告。
```

File: docusaurus/docs/using-https-in-development.md [hunk 5]
```diff
@@ -30,15 +30,15 @@ set HTTPS=true&&npm start
...
 
 ```json
 {
   "start": "HTTPS=true react-scripts start"
 }
 ```
 
-Or you can create a `.env` file with `HTTPS=true` set.
-[Learn more about environment variables in CRA](https://create-react-app.dev/docs/adding-custom-environment-variables).
+或者你可以使用 `HTTPS=true` 设置创建 `.env` 文件。
```
Comment (QC-L) [inline docusaurus/docs/using-https-in-development.md]:
```suggestion
或者你可以使用 `HTTPS=true` 创建 `.env` 文件。
```

File: docusaurus/docs/using-https-in-development.md [hunk 6]
```diff
@@ -30,15 +30,15 @@ set HTTPS=true&&npm start
...
+[了解有关CRA中环境变量的更多信息。](https://create-react-app.dev/docs/adding-custom-environment-variables).
```
Comment (QC-L) [inline docusaurus/docs/using-https-in-development.md]:
```suggestion
[了解有关 CRA 中环境变量的更多信息。](https://create-react-app.dev/docs/adding-custom-environment-variables).
```

**Comment by fikyair** [comment]
using-https-in-development 翻译修正

**Comment by fikyair** [comment]
update using-https-in-development

**Pull request merged**